roles/bwc/tasks/bwc_repos_cleanup_yum.yml

```diff
@@ -5,3 +5,6 @@
   yum_resository:
     name: "StackStorm_{{ bwc_repo }}"
     state: absent
+  tags:
+    - BWC repos
```

Comment:
Don't need `BWC repos` tag

Reply:
These tags orignally came from:

1. roles/bwc/tasks/bwc_repos_setup.yml: `tags` on "Handle bwc_license change" `include: license.yml`
2. roles/bwc/tasks/license.yml:  "Cleanup repo list file from disk"

Since `include_tasks` can't propagate tags until a later version of ansible, I manually propagated the tags.

If you want me to drop the ewc tag, that's fine.